request_a_govuk_domain/request/templates/registrar_details.html

```diff
@@ -13,6 +13,21 @@
 
```

Comment:
Can you use consistent indentation across the file? I think other templates use 2 characters.